Sources/Commands/SwiftTool.swift

```diff
@@ -232,6 +238,10 @@ public class SwiftTool<Options: ToolOptions> {
         self.buildPath = getEnvBuildPath() ??
             customBuildPath ??
             (packageRoot ?? currentWorkingDirectory).appending(component: ".build")
+        
+        if options.chdir != nil {
+            warning(message: "-C/--chdir is deprecated; use --package-path instead")
```

Comment:
Not related to this PR but we should think about adding deprecated support to the parser itself. That might be handy.